server/plugin/user.go

```diff
@@ -167,6 +167,9 @@ func (c *client) GetMe(mattermostUserID string) (*serializer.ServiceNowUser, err
 	if len(userDetails.UserDetails) == 0 {
 		return nil, fmt.Errorf("user doesn't exist on ServiceNow with email %s", mattermostUser.Email)
 	}
+	if len(userDetails.UserDetails) > 1 {
+		c.plugin.API.LogWarn("there are multiple users on the instance who have the same email address.", "Email", mattermostUser.Email)
```

Comment:
Here you can also add the instance name or URL whatever is there to show along with the email address
```suggestion
		c.plugin.API.LogWarn("multiple users with the same email address on instance.", "Email", mattermostUser.Email)
```